app/main.js

```diff
@@ -91,6 +92,23 @@ window.initApp = () => {
   }
 };
 
+const detection = new UnsupportedBrowserDetection();
+const checkSupportedBrowser = () => {
+  if (detection.isSupported()) {
+    setTimeout(() => {
+      window.initApp();
```

Comment:
This function doesn't need to be in `window` anymore. You can define a regular one.

Reply:
True - detached